packages/manager/modules/sign-up/src/form/activity/constants.js

```diff
@@ -67,6 +67,12 @@ export const COUNTRIES_VAT_LABEL = {
   MA: 'ICE',
 };
 
+export const SIRET_TRACKING_PREFIX = 'accountcreation::company-search';
+export const COMPANY_CREATED_PREFIX = `${SIRET_TRACKING_PREFIX}::company-created`;
+export const COMPANY_NOT_CREATED_PREFIX = `${SIRET_TRACKING_PREFIX}::company-not-created`;
+
 export default {
+  COMPANY_CREATED_PREFIX,
+  COMPANY_NOT_CREATED_PREFIX,
```

Comment:
```suggestion
  SIRET_TRACKING_PREFIX,
  COMPANY_CREATED_PREFIX,
  COMPANY_NOT_CREATED_PREFIX,
```

Reply:
Why should SIRET_TRACKING_PREFIX be exported ? 
It is only used in the constants file to build COMPANY_CREATED_PREFIX and COMPANY_NOT_CREATED_PREFIX

Reply:
You're exporting it during the definition as well.

`export const SIRET_TRACKING_PREFIX ...`